.eslintcache

```diff
@@ -1 +1 @@
,{"size":471,"mtime":1609946510661,"results":"18","hashOfConfig":"13"},{"size":525,"mtime":1610011574834,"results":"19","hashOfConfig":"13"},{"size":646,"mtime":1609973054544,"results":"20","hashOfConfig":"13"},{"size":318,"mtime":1609946510661,"results":"21","hashOfConfig":"13"},{"size":297,"mtime":1610011657049,"results":"22","hashOfConfig":"13"},{"size":196,"mtime":1610011551158,"results":"23","hashOfConfig":"13"},{"filePath":"24","messages":"25","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"ta6yyc",{"filePath":"26","messages":"27","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"29","messages":"30","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"31","messages":"32","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"33","messages":"34","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"35","messages":"36","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"37","messages":"38","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"39","messages":"40","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"41","messages":"42","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"28"},{"filePath":"43","messages":"44","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"45","messages":"46","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/index.jsx",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/reportWebVitals.js",[],["47","48"],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/App.jsx",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/Home.jsx",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/GameForum.jsx",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/styles/GameForumStyles.js",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/styles/HomeStyles.js",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/ForumCard.jsx",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/styles/ForumCardStyles.js",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/EventLink.jsx",[],"/Users/brianryan/Documents/Code/React/BettingForum/betting_forum/src/styles/EventLinkStyles.js",[],{"ruleId":"49","replacedBy":"50"},{"ruleId":"51","replacedBy":"52"},"no-native-reassign",["53"],"no-negated-in-lhs",["54"],"no-global-assign","no-unsafe-negation"]
```

Comment:
I don't know what the deal with this file is, I thought I'd added it to .gitignore but it still seems to be popping up. I'll look into it.